src/pyxdf/pyxdf.py

```diff
@@ -542,6 +604,157 @@ def _scan_forward(f):
             return False
 
 
+# Named tuple to represent monotonicity information.
+Monotonicity = namedtuple("Monotonicity", ["result", "n", "dec_count", "eq_count"])
```

Comment:
Does this need to be public? Also, and especially if this is just private, does this need to be a `NamedTuple`?